stdlib/public/Python/Python.swift

```diff
@@ -1088,14 +1116,16 @@ extension PartialRangeFrom : PythonConvertible where Bound : PythonConvertible {
     }
     self.init(lowerBound)
   }
+}
 
+extension PartialRangeUpTo : PythonConvertible where Bound : PythonConvertible {
   public var pythonObject: PythonObject {
     _ = Python // Ensure Python is initialized.
-    return Python.slice(lowerBound, Python.None, Python.None)
+    return Python.slice(Python.None, upperBound, Python.None)
   }
 }
 
-extension PartialRangeUpTo : PythonConvertible where Bound : PythonConvertible {
+extension PartialRangeUpTo : ConvertibleFromPython where Bound : ConvertibleFromPython {
```

Comment:
Fit within 80 columns.
```suggestion
extension PartialRangeUpTo : ConvertibleFromPython
  where Bound : ConvertibleFromPython {
```